roles/common/tasks/users.yml

```diff
@@ -32,3 +32,8 @@
     key: "{{ lookup('file', item.ssh_key) }}"
     state: "{{ item.state }}"
   with_items: "{{ sys_admins }}"
+
+- name: Remove Ferran's home directory
+  file:
+    path: /home/ferran
+    state: absent
```

Comment:
This is also needed because of that extra-step on app user? Our sysadmin role already deals with this.

Reply:
:woman_shrugging: I guess it's because it was still used as app user, yes

Reply:
I would create an issue to remove sys admins ability to log in as app user. My guess is it will clarify all these gotchas.

Not urgent of course...

Reply:
I don't really understand what you mean. The way we create the app user is copied from TO and has nothing too special: we create the user, in the group with its same name and add the SSH keys of the sysadmins to its authorized keys.

Reply:
The fact that we do that in TO doesn't imply that's correct. We stopped a while ago doing that since it's not needed anymore. You don't need to log in directly as `app_user`, we just log in with our personal user and then use `sudo su - app_user`.

As I said before, not urgent. It's just something that we don't do anymore on our machines, it's much easier to maintain.

Reply:
ok, I didn't know that was *the new way to go*